test/e2e/run_test.go

```diff
@@ -184,6 +184,41 @@ var _ = Describe("Podman run", func() {
 		Expect(session.OutputToString()).To(ContainSubstring("15"))
 	})
 
+	It("podman run blkio-weight-device test", func() {
+		session := podmanTest.Podman([]string{"run", "--rm", "--blkio-weight-device=/dev/zero:200", ALPINE, "cat", "/sys/fs/cgroup/blkio/blkio.weight_device"})
```

Comment:
I see not supported when trying to set this on /dev/zero on Fedora. Can we try with some other device like a hd?

Reply:
I don't know if we can guarantee device layout on CI, given we test on Travis, Papr, and also local Vagrant VMs. Can we make a loopback device and test with that?